src/e2e-test/resources/pluginParameters.properties

```diff
@@ -70,3 +70,16 @@ pubSubErrorThreshold=0
 pubSubStringValue=one
 pubSubNegativeValue=-100
 ## PUBSUBSINK-PLUGIN-PROPERTIES-END
+
+## BQEXECUTE-PLUGIN-PROPERTIES-START
+bqexecuteTableName=BQTable
+bqexecuteQuery=SELECT * FROM `cdf-athena.test_automation.demoCheck1`
+bqexecuteQueryUC=SELECT * FROM `cdf-athena.test_automation.demoCheck1` WHERE DATE(_PARTITIONTIME) = "2021-11-17" LIMIT 1000
+bqexecuteQueryUC1=SELECT * FROM `PROJECT_NAME.DATASET.TABLENAME`;
```

Comment:
bqexecuteQueryUC1 -> bqExecuteQuery

Only bqexecuteQueryUC1 is used in scenario. Remove all other unused queries

Reply:
@sawantpritam -done